docs/madr/decisions/071-transparent-proxy-config-map-handling-outside-the-control-plane.md

```diff
@@ -44,7 +44,14 @@ This inconsistency makes it hard to track where values are coming from.
 
 We stop using the transparent proxy config fields in the Dataplane object and mark them as deprecated. These fields will no longer be relied on by `kuma-sidecar` in Kubernetes deployments. Instead, the same information will be passed as metadata during the xDS connection, using values from the transparent proxy config passed via a CLI flag.
 
-Both `kuma-init` and `kuma-sidecar` will accept a `--config-dir` CLI flag. This directory will contain one or more configuration files, read in alphabetical order. In case of conflicts, values from the later file override earlier ones.
+`kuma-dp` will support a `--transparent-proxy` flag, and `kumactl install transparent-proxy` will support a unified `--config` flag. Both flags can accept one of the following values:
```

Comment:
Nitpick: WDYT about calling it `--transparent-proxy-config`?

```suggestion
`kuma-dp` will support a `--transparent-proxy-config` flag, and `kumactl install transparent-proxy` will support a unified `--config` flag. Both flags can accept one of the following values:
```

Reply:
:( I wish install transparent-proxy was in the DP instead of kumactl :(

Reply:
> Nitpick: WDYT about calling it --transparent-proxy-config?

I went with `--transparent-proxy` so that when no value is provided, it simply enables transparent proxy with the default config. Using `--transparent-proxy-config` makes that less intuitive. But we could support both and make one an alias of the other for flexibility. WDYT?

Reply:
@slonka I've updated the MADR and included my suggestion about aliases with an example